src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -497,7 +497,8 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
           }
         }
 
-        "should 200 OK if it has the full set of required attribute headers, plus optionals" in {
+        // TODO(dmohs): Flaky.
```

Comment:
Note: I intend to un-ignore these as part of this change as soon as I figure out why they are flaky.